readme.md

```
@@ -1,66 +1,139 @@
+node.get_info() -> Result<NodeInfo, ApiError> // returns NodeInfo and balances
 ```
 
 #### Channel Management
-```
-lni.fetch_channel_info()
+```rust
+// TODO - Not implemented
+node.channel_info()
 ```
 
 #### Event Polling
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)